tests/js/spec/stores/globalSelectionStore.spec.jsx

```diff
@@ -0,0 +1,13 @@
+import GlobalSelectionStore from 'app/stores/globalSelectionStore';
+
+describe('GlobalSelectionStore', function() {
+  it('get()', function() {
+    expect(GlobalSelectionStore.get()).toEqual({projects: []});
+  });
+
+  it('updateProjects()', function() {
+    expect(GlobalSelectionStore.get().projects).toEqual([]);
+    GlobalSelectionStore.updateProjects([1]);
```

Comment:
Using the action creator here would more accurately test how this store would be used in our application since we should never call the `GlobalSelectionStore.updateProjects` method directly.

Reply:
Updated to use this